README.md

```diff
@@ -19,44 +19,80 @@ We assume that you already have:
 
 Then, you can create an URS for your circuit in the following way:
 
-```rust,ignore
-use kimchi::{circuits::constraints, verifier::verify};
-use mina_curves::pasta::{fp::Fp, vesta::{Affine, VestaParameters}, pallas::Affine as Other};
+```rust
+use std::sync::Arc;
+use ark_ff::{FftField, PrimeField};
+use ark_poly::{EvaluationDomain, Radix2EvaluationDomain as D};
+use circuit_construction::*;
+use commitment_dlog::{commitment::CommitmentCurve, srs::SRS};
+use groupmap::GroupMap;
+use kimchi::verifier::verify;
+use mina_curves::pasta::{
+    fp::Fp,
+    vesta::{Affine, VestaParameters},
+};
 use oracle::{
-    constants::PlonkSpongeConstantsKimchi,
+    constants::*,
```

Comment:
Updated the `circuit-construction` to do the re-export. Please have a check if that makes sense, otherwise I can revert this commit and leave it to a dedicated future refactor task. 

btw, it seems to me it would look cleaner to have `circuit-construction/src/lib.rs` file separated into several different files and leave the `lib.rs` only holds the definitions of `mods` and `re-exports`.

Reply:
> btw, it seems to me it would look cleaner to have circuit-construction/src/lib.rs file separated into several different files and leave the lib.rs only holds the definitions of mods and re-exports.

agree, if you want to take a stab at it be my guest, otherwise I might get to it in a week or so

Reply:
I am happy to :D
I will create a PR for it.

Reply:
Here is an attempt to refactor the file structure. Let me know any suggestions :D https://github.com/katat/kimchi/pull/1